resources/apt/test/assets/apt_install_rolldice.dsc.yaml

```diff
@@ -0,0 +1,19 @@
+# Example to see if PowerShell 7 is installed, install it, or get all installed packages
+$schema: https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2024/04/config/document.json
+resources:
+- name: assertions
+  type: Microsoft.DSC/Assertion
+  properties:
+    $schema: https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2024/04/config/document.json
+    resources:
+    - name: os_check
+      type: Microsoft/OSInfo
+      properties:
+        family: Linux
+- name: apt_rolldice
+  type: DSC.PackageManagement/Apt
+  properties:
+    packageName: rolldice
+    _exist: true
```

Comment:
It would be better if this value was defined as a parameter so instead of having two similar config, you could just pass in `false` (and default to `true`).  See https://github.com/PowerShell/DSC/blob/main/dsc/tests/dsc_parameters.tests.ps1 for examples

Reply:
I'm happy to leave that as an exercise for the reader